src/Promitor.Core.Contracts/ResourceType.cs

```diff
@@ -31,6 +31,7 @@ public enum ResourceType
         EventHubs = 26,
         ExpressRouteCircuit = 27,
         ApplicationGateway = 28,
-        NetworkGateway = 29
+        NetworkGateway = 29,
+        Kubernetes = 30
```

Comment:
I'd prefer to rename the resource type of this scraper to `KubernetesService` to avoid confusion with vanilla Kubernetes.

Reply:
Do note that we should then also rename all our types from KubernetesXYZ to KubernetesServiceXYZ

Reply:
Sure, implemented in 87f02b321ff300fb0a6caf8aaa993365be42358a